create_key/create_key.go

```diff
@@ -0,0 +1,147 @@
+package createKey
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/NBISweden/sda-cli/helpers"
+	"github.com/elixir-oslo/crypt4gh/keys"
+	log "github.com/sirupsen/logrus"
+)
+
+// Help text and command line flags.
+
+// Usage text that will be displayed as command line help text when using the
+// `help encrypt` command
+var Usage = `
+USAGE: %s createKey <name> (-outdir <dirname>)
+
+createKey: Creates a crypt4gh encryption key pair, and saves it to
+           <name>.pub.pem, and <name>.sec.pem.
+           NOTE: keys created using this function should not be used when
+		         encrypting submission files, they should only be used for
+                 decrypting files downloaded from the archive.
```

Comment:
I think there is something off with the indentation here, second line of NOTE seems to have one additional space